docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md

```diff
@@ -67,7 +67,7 @@ stencil init [--url STORE_URL] [--token API_TOKEN]
 Starts the live theme preview using the theme files in the current directory.
 
 ```shell title="Usage: stencil start"
-stencil start [-V|--version] [-o|--open] [-v|--variation] [-t|--test] [-t|--tunnel]
+stencil start [-V|--version] [-o|--open] [-v|--variation] [-t|--test] [-t|--tunnel] [-nov|--no-verbose] [-vb|--verbose]
```

Comment:
```suggestion
stencil start [-V|--version] [-o|--open] [-v|--variation] [-t|--test] [-t|--tunnel] [-vb|--verbose]
```
We will provide `no-verbose` mode by default, so there is no need for this option any more.